app/controllers/integration_listener_controller.rb

```diff
@@ -14,7 +14,7 @@ def callback
     if @integration.save
       redirect_to app_path(state_app), notice: "Integration was successfully created."
     else
-      render :new, status: :unprocessable_entity
+      redirect_to app_integrations_path(state_app), notice: "Failed to create the integration -- #{@integration.errors.full_messages}"
```

Comment:
Can we show this properly? 

Maybe see if you can use `I18n.t('errors.messages.#{error_name}')` or something.

Reply:
There are no expected errors in this flow, I am trying to catch unexpected ones and show them to users. I am not sure I can do translation for unexpected API errors.

Reply:
On second thought, there are no model-level validations for connectable integrations. Any API exception will bubble up as an error anyway, so removing this.